### PR TITLE
Updates to README

### PR DESCRIPTION
**Pull request by mviitane**
**What this PR Includes**
- Minor changes to install tutorial links, description and features to keep README up to date
- Clean-up of some old information

**Fixes**
#876 

Signed-off-by: mviitanen <mviitanen@mirantis.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,119 +20,70 @@ k0s is an all-inclusive Kubernetes distribution with all the required bells and
...
-
 _We have seen a gap between the host OS and Kubernetes that runs on top of it: How to ensure they work together as they are upgraded independent from each other? Who’s  responsible for vulnerabilities or performance issues originating from the host OS that affect the K8S on top?_
 
-**&rarr;** k0s Kubernetes is fully self contained. It’s distributed as a single binary with no host OS deps besides the kernel. Any vulnerability or perf issues may be fixed in k0s Kubernetes.
+**&rarr;** k0s is fully self contained. It’s distributed as a single binary with no host OS deps besides the kernel. Any vulnerability or perf issues may be fixed in k0s Kubernetes.
 
 _We have seen K8S with partial FIPS security compliance: How to ensure security compliance for critical applications if only part of the system is FIPS compliant?_
 
-**&rarr;** k0s Kubernetes core + all included host OS dependencies + components on top may be compiled and packaged as a 100% FIPS compliant distribution with a proper toolchain.
+**&rarr;** k0s core + all included host OS dependencies + components on top may be compiled and packaged as a 100% FIPS compliant distribution with a proper toolchain.
```
Comment (kke) [inline README.md:47]:
```suggestion
**&rarr;** k0s core + all included host OS dependencies + components on top may be compiled and packaged as a 100% FIPS compliant distribution using a proper toolchain.
```

File: README.md [hunk 2]
```diff
@@ -21,119 +20,70 @@ k0s is an all-inclusive Kubernetes distribution with all the required bells and
...
 
 ## Scope
 
-While some Kubernetes distros package everything and the kitchen sink in, k0s tries to minimize the amount of "add-ons" to bundle in. Instead, we aim to provide robust and versatile "base" for running Kubernetes in various setups. Of course we will provide some ways to easily control and setup various "add-ons" but we will likely not bundle many of those into k0s itself. There's couple reasons why we think this is the correct way:
+While some Kubernetes distros package everything and the kitchen sink, k0s tries to minimize the amount of "add-ons" to bundle in. Instead, we aim to provide robust and versatile "base" for running Kubernetes in various setups. Of course we will provide some ways to easily control and setup various "add-ons", but we will not bundle many of those into k0s itself. There're couple of reasons why we think this is the correct way:
```
Comment (kke) [inline README.md:82]:
```suggestion
While some Kubernetes distros package everything and the kitchen sink, k0s tries to minimize the amount of "add-ons" to bundle in. Instead, we aim to provide robust and versatile "base" for running Kubernetes in various setups. Of course we will provide some ways to easily control and setup various "add-ons", but we will not bundle many of those into k0s itself. There are a couple of reasons why we think this is the correct way:
```

**Review by jasmingacic — APPROVED** [review]

**Review by kke — APPROVED** [review]

**Review by jasmingacic — COMMENTED** [review]
This is looking good. Just make sure to sing off your commits

**Comment by mviitane** [comment]
> This is looking good. Just make sure to sing off your commits

Thanks! I'll do that. Committing directly from the review suggestions didn't fully work as I expected.

**Pull request merged**